### PR TITLE
feat: send findings domain counts to frontend webview

### PR DESCRIPTION
**Pull request by ahtrotta**
Fixes applandinc/board#28

Makes the number of each type of finding available to the frontend

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -167,8 +167,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<AppMap
 
     let processService: NodeProcessService | undefined;
     let runtimeAnalysisCta: RuntimeAnalysisCtaService | undefined;
+    let projectState: ProjectStateService;
     {
-      const projectState = new ProjectStateService(
+      projectState = new ProjectStateService(
```
Comment (dustinbyrne) [inline src/extension.ts:172]: If we need `projectState` available to the outer scope, we can just drop the scope it's defined in.

File: src/appMapService.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { AppMapConfigWatcher } from './services/appMapConfigWatcher';
 import { AppmapUptodateService } from './services/appmapUptodateService';
 import Command from './services/command';
 import { NodeProcessService } from './services/nodeProcessService';
+import ProjectStateService, { ProjectStateServiceInstance } from './services/projectStateService';
```
Comment (dustinbyrne) [inline src/appMapService.ts:10]: `ProjectStateServiceInstance` looks to be unused

File: src/extension.ts (hunk 1 above)
Comment (ahtrotta) [reply src/extension.ts]: Okay, I dropped the scope. Why was it there in the first place?

File: src/appMapService.ts (hunk 2 above)
Comment (ahtrotta) [reply src/appMapService.ts]: I deleted it

File: test/fixtures/workspaces/project-several-findings/.nvmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+v16.15.0
```
[the remark this answers is not in the document]
Comment (ahtrotta) [reply test/fixtures/workspaces/project-several-findings/.nvmrc:1]: Deleted!

**Review by dustinbyrne — APPROVED** [review]

**Pull request merged**